### DISCUSSION TITLE
Is `sidecar` and `resouces` could be override?

### DISCUSSION THREAD

**Discussion by greenhat616** [tauri-apps/tauri#8368] [Q&A] [answered]
Hello, I wonder that `sidecar` or `resources` might follow different release step or frequency, and user might want to use the latest one(which is not bundled).
So is it possible to replace the binary in tauri app installed dir with new one?

**Comment by FabianLars** [comment]
Only on Windows if you use the perUser nsis/setup.exe installer. For everything else, no (including the .msi windows bundle and all bundle types on macos and linux).
And of course you'd have to do it yourself, there's no helper function for this.

**Reply by greenhat616** [reply]
Thanks for your reply. So it means **EXCEPT** nsis, I can safely override the files (in `resources` and `sidecar`) via Rust logic? 😃

**Reply by FabianLars** [reply, marked as answer]
No, it's the other way around. You can only do it with the nsis bundle.

The location of the sidecars/resources is read-only on all other methods, especially if we're talking about macos and linux.
